README.md

```diff
@@ -75,3 +75,15 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing
+
+To run tests run `poetry run pytest` 
+
+To run a single test file run `poetry run pytest <path_to_test_file>`
+
+To run a single test run `poetry run pytest -k <test_name>`
+
+## Linting
+
+Run `back .` before a commit to lint the code.
```

Comment:
Nice addition here 😄 I think a typo in that this should be `black .` as the command, but I'd actually recommend explicitly running it through poetry so you can be confident it will use their virtual environment
```suggestion
Run `poetry run back .` before a commit to lint the code.
```